### ISSUE TITLE
Перенести картинки из yandex.diskЗадача перенесена из TODO.

### ISSUE THREAD

**Issue by satandyh** [eucariot/SDSM#43]
Переносить копипастом через gitBook, что бы были масштабиремыми.

**Comment by JekRock** [comment]
Из Украины большинство картинок не открывается, так как yandex заблокирован

**Comment by eucariot** [comment]
Да, этот Issue как раз для того, чтобы перенести всё внутрь гитбука.

**Comment by mekatum** [comment]
В 3-й главе https://linkmeup.gitbook.io/sdsm/3.-static_routing/00-intervlan-routing#intervlan-routing не масштабируются картинки
